Listenbrainz/UI/Components/WelcomeView.swift

```diff
@@ -0,0 +1,38 @@
+
+
+// Listen-brainz icons
+
+
+
+import SwiftUI
+
+
+struct LBIcon1: Shape {
```

Comment:
Did you use chatgpt to write this? We can just use the svg icons directly by putting them in the assets as icons using some icon genertor online. You can find the respective logos here https://github.com/metabrainz/design-system/tree/master/brand/logos

Reply:
No, I've just converted the SVG code to Swift code using the online generator @akshaaatt

Reply:
Keep the images separate and don't get into using them as code. SVG is already enough I feel lol

Reply:
yep, sure :)